src/plugins/application_config/common/index.ts

```diff
@@ -0,0 +1,2 @@
+export const PLUGIN_ID = 'applicationConfig';
+export const PLUGIN_NAME = 'applicationConfig';
```

Comment:
If the value stored is the same, do we need 2 different variable names?

Reply:
Updated the value to be different.